.eslintrc.js

```diff
@@ -21,11 +22,32 @@ module.exports = {
     ecmaVersion: 12,
     sourceType: 'module',
   },
-  plugins: ['jest', 'react', '@typescript-eslint'],
+  plugins: ['jest', 'react', '@typescript-eslint', 'testing-library'],
   rules: {},
   settings: {
     react: {
       version: 'detect',
     },
   },
+  overrides: [
+    {
+      files: ['*.ts', '*.tsx'],
+      rules: {
+        'react/prop-types': 'off',
```

Comment:
prop-types aren't used in typescript.  I think this is raising an error just because we have a mix of js and ts in the project